lib/components/helpers/pretty-text-input.js

```diff
@@ -284,7 +284,10 @@ export default createReactClass({
     // Render read-only version.
     return (
       <div onKeyDown={this.onKeyDown} className={cx({'pretty-text-wrapper': true, 'choices-open': this.state.isChoicesOpen})}
-        onMouseEnter={this.switchToCodeMirror} onTouchStart={this.switchToCodeMirror}>
+        onMouseEnter={this.switchToCodeMirror}
+        onTouchStart={this.switchToCodeMirror}
+        onFocus={this.switchToCodeMirror}
```

Comment:
Have you tested if VoiceOver navigation triggers focus?

Reply:
@WestonThayer It will if you tab onto the field and then hit `ctrl + optn + shift + downarrow`, but the voiceover just calls it a 'group', which isn't very informative 😅
Adding `onFocus` made it so that when I tabbed onto the field it said 'text area' instead.

Reply:
`onFocus` sounds like a win! Thanks so much for figuring this out so quickly 😀 

I'd love to help QA this, would it be possible to deploy to a greek server or staging?

Reply:
@WestonThayer No problem! Hopefully it's the right fix, I'm not sure if we left `onFocus` off for a reason.

I'll ping you in slack as far as QAing this! 😄